eras/babbage/impl/cardano-ledger-babbage.cabal

```diff
@@ -65,7 +65,6 @@ library
     nothunks,
     plutus-ledger-api ^>= 1.0,
     plutus-tx ^>= 1.0,
-    set-algebra,
```

Comment:
I thought it was only me having problems with this, so I started to comment out ` -Wunused-packages` locally instead, oops :see_no_evil:

Reply:
yea, I think it's all of us :) your new nix prompt is really nice, btw, highlighting the errors :)

Reply:
Glad it's useful! :tada: